### PR TITLE
Adding logging feature to a file

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
I make container log enabled to a file /home/azureuser/logs/teamdeploy.out

Once I success the whole deployment test, I'll share with you.

### CODE REVIEW

File: provision-vm/Docker/Readme.md [hunk 1]
```diff
@@ -12,13 +12,11 @@ For getting started, create a docker daemon using the docker image. For experime
...
 
 ```
-$ docker run -d --name some-docker --privileged docker:stable-dind
 $ docker run --link some-docker:docker -it proctorvm /bin/bash
-root@531ed021c2c5:/home/azureuser# export DOCKER_HOST='tcp://docker:2375'
 root@531ed021c2c5:/home/azureuser# export AZUREUSERNAME=YOUR_SERVICE_PRINICPAL_NAME
 root@531ed021c2c5:/home/azureuser# export AZUREPASSWORD=YOUR_SERVICE_PRINICPAL_PASSWORD
 root@531ed021c2c5:/home/azureuser# export SUBID=YOUR_SUBSCRIPTION_ID
-root@531ed021c2c5:/home/azureuser# export LOCATION=westus2
+root@531ed021c2c5:/home/azureuser# export LOCATION=westus2s
```
Comment (dcaro) [inline provision-vm/Docker/Readme.md]: is this a typo?

Comment (TsuyoshiUshio) [reply provision-vm/Docker/Readme.md]: typo. Thanks. I fixed.

**Comment by dcaro** [comment]
Waiting on deployment to confirm merge.

**Comment by TsuyoshiUshio** [comment]
It works. I can see

Download your credentials for the DevOps OpenHack

A zip file with the credentials 
Enjoy !

Now.

**Comment by TsuyoshiUshio** [comment]
@dcaro Merge it! :)

**Comment by dcaro** [comment]
Let me know when I can delete the branch.

**Comment by TsuyoshiUshio** [comment]
You can delete the branch.

**Comment by dcaro** [comment]
This line will break is the branch is deleted: https://github.com/Azure-Samples/openhack-devops-proctor/blob/master/provision-vm/azuredeploy.json#L242

**Pull request merged**